gnucash/gtkbuilder/dialog-import.glade

```diff
@@ -945,6 +945,21 @@
             <property name="position">1</property>
           </packing>
         </child>
+        <child>
+            <object class="GtkCheckButton" id="reconcile_after_close_button">
+                <property name="label" translatable="yes">Reconcile after match</property>
+                <property name="visible">True</property>
```

Comment:
Have you tried setting the default visibility to False ? If I'm not mistaken gtk_widget_show_all won't automatically display such widgets. That may make it no longer necessary to call `gnc_gen_trans_list_show_reconcile_after_close` in `gnc_gen_trans_list_show_all`.

Reply:
> Have you tried setting the default visibility to False

That was the first thing I tried, and it didn't work. But I'll try again. It's quite possible that I goofed something the first time around (perhaps didn't build ALL).

> Do you plan to change this still ?

I had fixed that but it looks like I lost it when I rebased against master. Let me put it back, sorry about that.

Reply:
For information, what you needed to do was set the visibility of the button to false and also enable the setting to ignore gtk_widget_show_all